roles/zabbix_proxy/tasks/Debian.yml

```diff
@@ -206,9 +227,9 @@
   when:
     - zabbix_proxy_database == 'mysql'
     - zabbix_proxy_install_database_client
-    - ansible_distribution_release == "stretch"
+    - ansible_distribution_release != "buster"
```

Comment:
Wouldn't this also impact debian 11? Or is Deb9 setup similar to Deb11? (sorry I am absolutely out of the loop when it comes to this at the moment)

Reply:
Again I followed the server role commands there. To be honest, meta-package default-mysql-client is present in all 3 supported Debian distros. I can't figure out why the mariadb-client is used instead at Deb10 below

Reply:
Honestly not sure what this is going to do here and right now under the current 1.X branch we're not testing for most of the older OSs.

Reply:
Yeah, ok. I wouldn't dissect this anymore.

For me this is OK to go. @pyrodie18 Feel free to merge if you have no further requests.